### ISSUE TITLE
Include code block highlighting themes

### ISSUE THREAD

**Issue by ianchanning** [davidgranstrom/nvim-markdown-preview#4]
Pandoc has support for [code block syntax highlighting themes](https://pandoc.org/MANUAL.html#syntax-highlighting).

The ones that pandoc provides are very limited (`pandoc --list-highlight-styles`), but they are based on the [Kate editor syntax colour themes](https://github.com/KDE/syntax-highlighting#color-theme-files) and can be extended if you include  the Kate editor themes in your plugin and allow an option for setting one. 

You can download all the theme files, they're just JSON, from here: https://github.com/KDE/syntax-highlighting/tree/master/data/themes

For example I used the `solarized-dark.theme` then you can set the them using `--highlight-style=solarized-dark.theme`, so my full command is now (I also downloaded your `solarized-dark.css` locally for testing:

```sh
pandoc -f markdown test.md -o test.html --standalone -t html --katex --include-in-header solarized-dark.css --highlight-style=solarized-dark.theme
```

If you need some markdown to test with I've been using some tutorial notes I wrote: https://github.com/ianchanning/neural-network-js/blob/master/tutorial/slides.tex.md 

This is a side-by-side comparison of your default output and from pandoc with `--katex` and `--highlight-style=solarized-dark.theme` - compare the readability of the code blocks.

![Screenshot from 2021-11-14 20-20-37](https://user-images.githubusercontent.com/233318/141695413-025e96f1-78c4-4a8f-8321-b3374792776e.png)

**Comment by davidgranstrom** [comment]
@ianchanning Thanks for this suggestion, the highlighting looks great! We could definitely add an option for setting `highlight-style`, and even copy the themes and add them to the repo to make it easier to switch styles. Maybe the default would be to match the selected theme with the highlight style as in your example.

I would be open to PR for this feature too as with #3  :) I could also take a look at it later, but at the moment I'm a bit busy with other obligations.

**Comment by ianchanning** [comment]
@davidgranstrom To add this I would have to modify the interface to `Pandoc.generate(css, restart)` I think. I would propose that I add a `highlight` option: `Pandoc.generate(css, highlight, restart)` - does that seem ok to you?

**Comment by ianchanning** [comment]
@davidgranstrom Rather than have separate highlight options I have called the 'css' parameter 'theme' now and then based off of that generate a css file path and add a new `highlight` directory with `*.theme` files that match the css names.

**Comment by davidgranstrom** [comment]
Resolved with #6
